index.js

```diff
@@ -531,23 +531,6 @@ RingPop.prototype.onMemberFaulty = function onMemberFaulty(member) {
     this.suspicion.stop(member);
 };
 
-RingPop.prototype.onMemberJoined = function onMemberJoined(member) {
-    this.stat('increment', 'membership-update.new');
-    this.logger.debug('member has joined', {
-        local: this.membership.localMember.address,
-        joined: member.address
-    });
-
-    this.ring.addServer(member.address);
```

Comment:
So basically; when a member joins we should not add it to the server.

We should add it when we hear from it on the first ping or something ?